### PR TITLE
register new flutter_boost

### PR DESCRIPTION
**Pull request by noborder**
register new flutter_boost

### CODE REVIEW

**Comment by noborder** [comment]
@googlebot I signed it!

**Comment by noborder** [comment]
why tests Failing  ?
The local execution command succeeded.
command:
 ../flutter/bin/cache/dart-sdk/bin/dart ../flutter/dev/customer_testing/run_tests.dart --skip-template --verbose registry/*.test

**Comment by noborder** [comment]
| 00:07 +22: /private/var/folders/4g/h1k_8rkn0sz1rj6fn6_21tl80000gp/T/flutter_customer_testing.flutter_boost.euwoJr/tests/test/lib/unit/container_manager_test.dart: ... based on the number of pages
| 00:07 +22: All tests passed!
Tests finished.
Deleting temporary directory...

**Comment by noborder** [comment]
@goderbauer

**Comment by goderbauer** [comment]
Take a look at the log: https://api.cirrus-ci.com/v1/task/5289922932244480/logs/main.log

Looks like there are some compiler errors in your code:

```
| Compiler message:
| ../lib/container/boost_container.dart:202:22: Error: This expression has type 'void' and can't be used.
|         return super.pop<T>(result);
|                      ^
```

The return type of pop recently changed to void as is detailed in this migration guide here: https://flutter.dev/docs/release/breaking-changes/route-navigator-refactoring

**Comment by noborder** [comment]
> Take a look at the log: https://api.cirrus-ci.com/v1/task/5289922932244480/logs/main.log
> 
> Looks like there are some compiler errors in your code:
> 
> ```
> | Compiler message:
> | ../lib/container/boost_container.dart:202:22: Error: This expression has type 'void' and can't be used.
> |         return super.pop<T>(result);
> |                      ^
> ```
> 
> The return type of pop recently changed to void as is detailed in this migration guide here: https://flutter.dev/docs/release/breaking-changes/route-navigator-refactoring

thank you

**Comment by noborder** [comment]
test

**Comment by noborder** [comment]
@goderbauer

**Comment by noborder** [comment]
ERROR: Failed to fetch repository.
Contact: raboughanem@google.com
1 failure.

**Comment by goderbauer** [comment]
Looks like there's something wrong with the git hash you provided: https://github.com/flutter/tests/pull/25/checks?check_run_id=497649312

**Comment by noborder** [comment]
> Looks like there's something wrong with the git hash you provided: https://github.com/flutter/tests/pull/25/checks?check_run_id=497649312

It's not me, it's flutter_samples_gallery.test

**Comment by noborder** [comment]
@rami-a

**Comment by noborder** [comment]
@rami-a 


++ dart flutter/dev/customer_testing/run_tests.dart --repeat=15 --skip-template registry/DanTup_tiler.test registry/flutter_boost.test registry/flutter_packages.test registry/flutter_samples_gallery.test registry/flutter_svg.test registry/provider.test registry/rainbowmonkey.test registry/template.test
>> git -c core.longPaths=true -C tests checkout 3a05f1f286c99a99a40c2035fd17488e9b654ba5
| fatal: reference is not a tree: 3a05f1f286c99a99a40c2035fd17488e9b654ba5
ERROR: Failed to fetch repository.
Contact: raboughanem@google.com

**Comment by noborder** [comment]
@goderbauer

**Comment by goderbauer** [comment]
I restarted all the tests, let's see what happens.

**Comment by noborder** [comment]
> I restarted all the tests, let's see what happens.

 what happens?

**Comment by goderbauer** [comment]
https://github.com/flutter/tests/pull/28 should fix it. Once that is submitted, you'll need to rebase this PR.

**Comment by noborder** [comment]
@goderbauer

File: registry/flutter_boost.test [hunk 1]
```diff
@@ -0,0 +1,6 @@
+contact=noborder@qq.com
+fetch=git clone https://github.com/alibaba/flutter_boost.git tests
+fetch=git -C tests checkout 4aa4a9ec69cf912075083d695a55313ee06bb38e
+update=.
+# test=flutter analyze
```
Comment (Hixie) [inline registry/flutter_boost.test]: there's a lot of value in analyzing the code as well if possible

**Pull request merged**